extensions/vscode/lsp/language-configuration.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
This file was taken from `share/doc/wake/syntax/vscode`. I think you should remove `share/doc/wake/syntax/vscode` as part of this PR (which will then also mark this file as moved instead of freshly created). When you remove `share/doc/wake/syntax/vscode`, please also move the README.md (and perhaps enhance it).

Reply:
Done.